public/robots.txt

```diff
@@ -3,6 +3,17 @@
 # To ban all spiders from the entire site uncomment the next two lines:
 # User-Agent: *
 # Disallow: /
+
+User-agent: SemrushBot
+Disallow: /
+
+User-agent: AhrefsBot
+Disallow: /
+
+User-agent: *
+Disallow: /snps
```

Comment:
Should we keep `/snps` as it contains information people potentially search for?

If anyone has the time, we could try to figure out how much search traffic comes from searches for SNPs. :wink:

Reply:
Yes, let's keep `/snps`